specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip/checkRouting/{targetNumber}": {
+      "get": {
+        "tags": [
+          "NumberRoutingValidation"
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckRoutingForNumber",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "targetNumber",
+            "description": "Phone number to test routing patterns against",
+            "required": true,
+            "type": "string",
+            "x-example": "+11234567890"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "$select",
```

Comment:
This should be named `select`, not `$select`: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options

Reply:
Fixed